site/app/controllers/MiscController.php

```diff
@@ -427,17 +416,7 @@ public function downloadSubmissionZip($gradeable_id, $user_id, $version, $is_ano
                     $file_path = $file->getRealPath();
                     $relative_path = substr($file_path, strlen($path) + 1);
                     if ($this->core->getAccess()->canI("path.read", ["dir" => $folder_name, "path" => $file_path, "gradeable" => $gradeable, "graded_gradeable" => $graded_gradeable, "gradeable_version" => $gradeable_version->getVersion()])) {
-                        // For scanned exams, the directories get polluted with the images of the split apart
-                        // pages, so we selectively only grab the PDFs there. For all other types,
-                        // we can grab all files regardless of type.
-                        if ($gradeable->isScannedExam()) {
-                            if (mime_content_type($file_path) === 'application/pdf') {
-                                $zip_stream->addFileFromPath($folder_name . "/" . $relative_path, $file_path);
-                            }
-                        }
-                        else {
-                            $zip_stream->addFileFromPath($folder_name . "/" . $relative_path, $file_path);
-                        }
+                        $zip_stream->addFileFromPath($folder_name . "/" . $relative_path, $file_path);
```

Comment:
It might be better to use `FileUtils::joinPaths()` here instead of concatenating strings manually.